djaoapp/static/js/djaoapp-login.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const passwordFields = document.querySelectorAll('input[name="password"], ' +
+      'input[name="new_password"], input[name="new_password2"]');
+  passwordFields.forEach(field => {
```

Comment:
That's an interesting approach. I would have put the HTML in [_form_fields.html](https://github.com/djaodjin/djaoapp/blob/36147ce13c6b8dd89ee30499d391104d5cbdbba5/djaoapp/templates/jinja2/_form_fields.html#L180) and toggle `display: block;` / `display: none` in the Javascript.